pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -118,10 +148,25 @@ func (cache *claimInfoCache) hasPodReference(UID types.UID) bool {
 	defer cache.RUnlock()
 
 	for _, claimInfo := range cache.claimInfo {
-		if claimInfo.podUIDs.Has(string(UID)) {
+		if claimInfo.PodUIDs.Has(string(UID)) {
 			return true
 		}
 	}
 
 	return false
 }
+
+func (cache *claimInfoCache) syncToCheckpoint() {
+	cache.RLock()
+	defer cache.RUnlock()
+
+	claimInfoStateList := make(state.ClaimInfoStateList, 0)
```

Comment:
If we prefer to use `make` here would it make sense to set an expected capacity and avoid calling `append` below?

Reply:
yeah, either stop using `append()` or turn this into a `var claimInfoStateList state.ClaimInfoStateList`

Reply:
sure will fix

Reply:
not doing append would be a bit faster, I think.

Reply:
according to stackoverflow they are almost the same from go 1.10 

https://stackoverflow.com/questions/38654729/golang-slice-append-vs-assign-performance

Reply:
only with small sizes, I believe. It's probably due to some kind of pre-allocation, but generally re-slicing is longer operation than assignment. You can read about this in the comments on stackoverflow.

Reply:
I don't have a preference here. Whatever you decide is fine so long as it is correct.